From e3f05382c7abaefdff8a62a8572f499ca56239f7 Mon Sep 17 00:00:00 2001
From: Yuxi Sun <b36102@freescale.com>
Date: Fri, 10 Aug 2012 15:33:00 +0800
Subject: ENGR00220038-2 fix Kconifg waring

fix Kconfig waring of defaults for choice values not supported

Signed-off-by: Yuxi Sun <b36102@freescale.com>
---
 drivers/media/video/mxc/capture/Kconfig |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/media/video/mxc/capture/Kconfig b/drivers/media/video/mxc/capture/Kconfig
index 17fdd36..a7b14cb 100644
--- a/drivers/media/video/mxc/capture/Kconfig
+++ b/drivers/media/video/mxc/capture/Kconfig
@@ -130,7 +130,6 @@ choice
 config MXC_IPU_DEVICE_QUEUE_SDC
 	tristate "Queue ipu device for overlay library"
 	depends on VIDEO_MXC_IPU_CAMERA
-	default y
 	---help---
 	  Use case CSI->MEM->IPU DEVICE->SDC:
 		Images from sensor will be frist recieved in memory,then
@@ -140,7 +139,6 @@ config MXC_IPU_DEVICE_QUEUE_SDC
 config MXC_IPU_PRP_VF_SDC
 	bool "Pre-Processor VF SDC library"
 	depends on VIDEO_MXC_IPU_CAMERA
-	default n
 	---help---
 	  Use case PRP_VF_SDC:
 		Preprocessing image from smart sensor for viewfinder and
-- 
1.7.9.5

